backend/src/v2/cmd/launcher-v2/main.go

```diff
@@ -43,6 +44,7 @@ var (
 	mlmdServerPort    = flag.String("mlmd_server_port", "8080", "The MLMD gRPC server port.")
 	logLevel          = flag.String("log_level", "1", "The verbosity level to log.")
 	publishLogs       = flag.String("publish_logs", "true", "Whether to publish component logs to the object store")
+	cacheEnabledFlag  = flag.String("cache_enabled", "", "Enable cache globally.")
```

Comment:
I don't want `ParseBool` to fail if the flag isn't provided.
```suggestion
	cacheEnabledFlag  = flag.String("cache_enabled", "true", "Enable cache globally.")
```

Reply:
This is not user-facing. Don't you think it's dangerous to default a boolean by mistake?

Reply:
Either way is fine for me. This comment is depended on for this https://github.com/kubeflow/pipelines/pull/11831#discussion_r2066847892 comment.

@HumairAK do you have a preference?